docs/meetings/2023/TechArea20230829.md

```diff
@@ -31,10 +31,23 @@ Triage duty shifts Tue-Mon
 OSG 23:
 
 -   AI (Matt): Update mash scripts to merge OSG and UW HTCondor repos (SOFTWARE-5653)
+    - Table in design doc for which htcss repos to merge with osg
+-   AI (Matt): Complete the two -contrib builds (reach out to Mat for what to do with repo) 
 -   AI (Matt): Kick off OSG 23 XRootD tests (worker node and XRootD)
+    - Most tests require the OSG-23 condor build, these two don't
+    - May require slight updates to test repo
```

Comment:
Namely something in this miserable mess of regexes https://github.com/opensciencegrid/vm-test-runs/blob/master/bin/vmu.py#L96-L119. Should be easy enough to just kick off OSG 23 tests when osg-release is ready and see how the VMU test infrastructure fares